app/workers/rate_limit_checker_worker.rb

```diff
@@ -1,3 +1,4 @@
+# TODO: remove this when Sidekiq has exhausted these workers
```

Comment:
I think it might be nice to add a team name or a github issue link here to the `TODO`.
It would ensure that we know who is planning on doing it and will make it easier to `grep` for in the codebase.

Just a suggestion; we've adopted this on @thepracticaldev/delightful, and it has been working pretty well!!

Reply:
done, thanks!